velox/exec/TableScan.h

```diff
@@ -49,6 +49,16 @@ class TableScan : public SourceOperator {
       column_index_t outputChannel,
       const std::shared_ptr<common::Filter>& filter) override;
 
+  void close() override {
+    if (splitPreloader_) {
+      for (auto split : preloadingSplits_) {
+        split->dataSource->close();
+      }
+      preloadingSplits_.clear();
+    }
+    Operator::close();
```

Comment:
Why call `Operator::close()` here explicitly?

Reply:
Because here we override the parent class' close method, and we need to call it in override method to release other resources.
Thanks.